microsoft-edge/webdriver-chromium/release-signing.md

```diff
@@ -0,0 +1,56 @@
+---
+description: Information about GPG signing for Selenium Tools for Microsoft Edge releases.
+title: Selenium Tools Release Signing
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 01/22/2021
+ms.topic: article
+ms.prod: microsoft-edge
+ms.technology: devtools
+keywords: microsoft edge, web development, html, css, javascript, developer, webdriver, selenium, testing, tools, automation, test
+---
+# Selenium Tools release signing  
+
+This article provides the public signing key for [Selenium Tools for Microsoft Edge][GithubMicrosoftEdgeSeleniumTools] releases.  Use the following keys to validate packages provided by the [Microsoft Edge Developer Tools team][TwitterEdgeDevTools].  

```

Comment:
Adding a tutorial link on verifying PGP signatures seems out of scope for learning how to automate Edge. This is meant as a reference rather than a guide.

Reply:
Made the suggested text changes.